.github/workflows/ci.yml

```
@@ -0,0 +1,209 @@
+
+      - name: 🧪 Run Unit Tests
+        run: flutter test test/features/maze_ball/ --coverage --reporter expanded
+
+      - name: 📊 Generate Coverage Report
+        run: |
+          sudo apt-get update
+          sudo apt-get install -y lcov
+          genhtml coverage/lcov.info -o coverage/html
+
+      - name: 📤 Upload Coverage to Codecov
+        uses: codecov/codecov-action@v3
```

actionlint (1.7.7)

the runner of "codecov/codecov-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)